docs/encrypted_images.md

```diff
@@ -1,22 +1,22 @@
 <!--
-#
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-#  KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-#
+    #
```

Comment:
I do see an inconsistency as to the formatting of the license text. It's fine to fix these to be more consistent (although I would prefer the '-' prefix, since it matches the xml comment header. But, a fix to the license text formatting should be in a separate patch.

Reply:
I do not really know where the inconsistency is. Could you poinz out the problematic parts?
(The 4 space indentation is required for the correct rendering.)

Reply:
Just that one of the doc files uses a "-" prefix on the license block, and other files use a "#" prefix. I don't really understand the "#" prefix, as that is not a comment character is Markdown.

Reply:
Regardless, we need to split the change to the license text to a separate commit. I don't really like using the '#' as a comment character, since I don't think it is meaningful to markdown. Where is the license block showing up that we don't want it?

Reply:
I created a separate PR for this change: #721

Reply:
Grumble. Markdown is intentionally poorly specified, and apparently this kind of behavior is John Gruber's intent. At some point, we really do need to decide what markdown processor(s) matter. In this case, the docs are rendered with github's built-in renderer, as well as Jekyll for the website.

Reply:
I can absolutely accept, if we decide local/offline Markdown viewers are not a priority. Probably most people who have the source, will view the raw Markdown files in a text editor, not in a reader.